lib/ts/recipe/session/cookieAndHeaders.ts

```diff
@@ -152,17 +146,8 @@ export function setToken(
             expires,
             tokenType === "refresh" ? "refreshTokenPath" : "accessTokenPath"
         );
-
```

Comment:
there is a comment above "We intentionally use accessTokenPath for idRefresh tokens" -> remove that